src/main/java/net/minecraftforge/fmllegacy/packs/ResourcePackLoader.java

```diff
@@ -60,7 +60,7 @@ public static void loadResourcePacks(PackRepository resourcePacks, BiFunction<Ma
     }
 
     public static List<String> getPackNames() {
-        return ModList.get().applyForEachModFile(mf->"mod:"+mf.getModInfos().get(0).getModId()).collect(Collectors.toList());
+        return ModList.get().applyForEachModFile(mf->"mod:"+mf.getModInfos().get(0).getModId()).filter(n->!n.equals("mod:minecraft")).collect(Collectors.toList());
```

Comment:
What does this fix?

Reply:
A logged error saying the datapack mod:minecraft is missing, this filtering was removed during the port to 1.17, not sure why.
vanilla resources are already loaded separately from this pack.